go-controller/hybrid-overlay/pkg/controller/node_windows.go

```diff
@@ -1,17 +1,41 @@
 package controller
 
 import (
+	"bytes"
+	"encoding/json"
+	"net"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/hybrid-overlay/pkg/types"
 	houtil "github.com/ovn-org/ovn-kubernetes/go-controller/hybrid-overlay/pkg/util"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/factory"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kube"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
 
 	kapi "k8s.io/api/core/v1"
 	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog"
+
+	"github.com/Microsoft/hcsshim/hcn"
+)
+
+const (
+	// Hard-coded constants
+	useAutomaticDns   = true                         // If true, the Windows host DNS is used to resolve container DNS requests
+	networkName       = "OpenShiftNetwork"           // In practice, this is the virtual switch name
+	baseNetworkName   = "BaseOpenShiftNetwork"       // Name of the base network
+	fakeSubnetCIDR    = "11.0.0.0/30"                // We just need a subnet to create the base network. This subnet is actually invisible to the PODs/Nodes.
+	fakeSubnetGateway = "11.0.0.2"                   // Gateway IP for the fake Subnet. This gateway is invisible to the PODs/Nodes.
```

Comment:
not suitable default values

Reply:
@danwinship Jocelyn confirms that these values are not actually used for anything in the base network, and that pods could still talk to 11/8 if they needed to. But he also said it's OK to switch to something private so I changed to 10.64/16